x-pack/platform/packages/shared/kbn-elastic-assistant-common/impl/schemas/attack_discovery/crud_attack_discovery_schedules_route.schema.yaml

```diff
@@ -0,0 +1,218 @@
+openapi: 3.0.0
+info:
+  title: Attack discovery scheduling API endpoint
+  version: '1'
+components:
+  x-codegen-enabled: true
+  schemas:
+    AttackDiscoveryGenericResponse:
+      type: object
+      additionalProperties: true
+      description: Object containing Attack Discovery schedule.
+    AttackDiscoveryGenericError:
+      type: object
+      description: An attack discovery generic error
+      properties:
+        statusCode:
+          type: number
+        error:
+          type: string
+        message:
+          type: string
+
+paths:
+  /internal/elastic_assistant/attack_discovery/schedules:
+    post:
+      x-codegen-enabled: true
+      x-labels: [ess, serverless]
+      operationId: CreateAttackDiscoverySchedules
+      description: Creates attack discovery schedule
+      summary: Creates attack discovery schedule via the Elastic Assistant
+      tags:
+        - attack_discovery_schedule
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: './schedules.schema.yaml#/components/schemas/AttackDiscoveryScheduleCreateProps'
+      responses:
+        200:
+          description: Successful response
+          content:
+            application/json:
+              schema:
+                $ref: './schedules.schema.yaml#/components/schemas/AttackDiscoverySchedule'
+        400:
+          description: Bad request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/AttackDiscoveryGenericError'
+
+  /internal/elastic_assistant/attack_discovery/schedules/{id}:
+    get:
+      x-codegen-enabled: true
+      x-labels: [ess, serverless]
+      operationId: GetAttackDiscoverySchedules
+      description: Gets attack discovery schedule
+      summary: Gets attack discovery schedule via the Elastic Assistant
```

Comment:
> Gets attack discovery schedule via the Elastic Assistant

nit: I'm wondering if _via the Elastic Assistant_ is applicable in this summary (and the others in this route)

Reply:
done in https://github.com/elastic/kibana/pull/216656/commits/89adba597b631e0c7d4a055927a7fb2543cc61b7